example/js/example_settings.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
+class ExampleSettings extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.onBlurSelection = this.onBlurSelection.bind(this);
```

Comment:
https://babeljs.io/blog/2015/06/07/react-on-es6-plus -- look at the section on arrow functions to simplify this